src/view/text.rs

```diff
@@ -43,13 +44,13 @@ impl<T, A> View<T, A> for String {
         }
     }
 
-    fn event(
+    fn message(
         &self,
-        _id_path: &[crate::id::Id],
+        _id_path: &[xilem_core::Id],
         _state: &mut Self::State,
-        _event: Box<dyn Any>,
+        _message: Box<dyn std::any::Any>,
         _app_state: &mut T,
-    ) -> MessageResult<A> {
-        MessageResult::Stale
+    ) -> xilem_core::MessageResult<A> {
+        xilem_core::MessageResult::Nop
```

Comment:
One more (nit-picky) thing, this should be `Stale`, as events should not reach this, `Nop` should AFAIK only be used, when this view *expects* an event.

Reply:
Oh, just noticed, that I have copied that in my string extension PR as well, so that applies to the other String View impls too I guess...

Reply:
Not sure I really understand why this is, but I shall take your word for it (+ I can see other code doing this).